pl_bolts/models/autoencoders/basic_vae/basic_vae_module.py

```diff
@@ -77,23 +77,23 @@ def __init_system(self):
         self.encoder = self.init_encoder()
         self.decoder = self.init_decoder()
 
-    def __set_pretrained_dims(self, pretrained):
-        if pretrained == 'imagenet2012':
-            self.datamodule = ImagenetDataModule(data_dir=self.hparams.data_dir)
```

Comment:
This datamodule is never used, eh? I don't see the point of this function

Reply:
@ananyahjha93 ?